src/commands/nollegrupp/subcommands/add/nollegrupp-add.handler.ts

```diff
@@ -8,12 +8,20 @@ export async function handleNollegruppAdd(
 	const name = interaction.options.getString(NollegruppAddVariables.NAME);
 	const code = interaction.options.getString(NollegruppAddVariables.CODE);
 
+	let result = true;
```

Comment:
```suggestion
	let result = false;
```
it shouldn't happen, but if it does, it's better to show an error than success